src/utils/hooks/useStorageRequestParams.ts

```diff
@@ -0,0 +1,21 @@
+import {useMemo} from 'react';
+
+import {EVersion} from '../../types/api/storage';
+import {USE_BACKEND_PARAMS_FOR_TABLES_KEY} from '../constants';
+import {useSetting} from './useSetting';
+
+export const useStorageRequestParams = ({filter}: {filter?: string}) => {
+    const [useBackendParamsForTables] = useSetting<boolean>(USE_BACKEND_PARAMS_FOR_TABLES_KEY);
+
+    // If backend params are enabled, update params value to use them in fetch request
+    // Otherwise no params will be updated, no hooks that depend on requestParams will be triggered
+    return useMemo(() => {
+        if (useBackendParamsForTables) {
+            return {
+                version: EVersion.v2,
```

Comment:
Currently `v2` storage has some issues, so it will be used only when params enabled, as only `v2` works with sorting and filters